src/clojure.grammar

```diff
@@ -109,7 +144,10 @@ Operator { !operator Symbol }
     (!["] | "\\" _)+
   }
 
-  Character { "\\" (std.asciiLetter | std.digit | "@")+ }
+  multiChar { ("u" std.digit+) | ("o" std.digit+) | (std.asciiLetter std.asciiLetter+) }
+  singleChar { ![\n\r\s] }
```

Comment:
I am not sure if this is a good/correct way to say "not whitespace". I get an "overlapping character range" error if I use `![std.whitespace]`, even with the precedence rule below.